leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -590,7 +592,7 @@
   /**
    * This returns the references in the leak path that should be ignored by all on Android.
    */
-  public static ExcludedRefs.Builder createAndroidDefaults() {
+  @NonNull public static ExcludedRefs.Builder createAndroidDefaults() {
```

Comment:
nit: for return values, apply @NonNull and @Nullable just before the type, i.e., `public static @NonNull ExcludedRefs.Builder`.